Apigee/Mint/DeveloperRatePlan.php

```diff
@@ -132,6 +138,15 @@ public function loadFromRawData($data, $reset = false)
         if ($reset) {
             $this->initValues();
         }
+
+        if (isset($data['ratePlan']) && is_array($data['ratePlan']) && count($data['ratePlan']) > 0) {
```

Comment:
Rate Plan must be set first because all date property setters are using this object to retrieve the current timezone of the organization.